src/nvim/eval.c

```diff
@@ -15140,8 +15140,11 @@ static void f_sockconnect(typval_T *argvars, typval_T *rettv, FunPtr fptr)
     return;
   }
 
+  snprintf((char *)IObuff, sizeof(IObuff), "%s:%" PRIdLINENR,
```

Comment:
shouldn't this be a function? Also channel can call it, rather than needing to modify all entry points in eval.c